### PR TITLE
Update kubeadm reference for 1.23

### PR DESCRIPTION
**Pull request by tengqm**
This PR refresh the generated files for kubeadm v1.23.

### CODE REVIEW

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset_phase_update-cluster-status.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
-<!--
+{{<note>}}
```
Comment (jihoon-seo) [inline content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset_phase_update-cluster-status.md:1]: [Netlify build fails](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/61b2a26a35b0b20007d6d0f5#L76).

> Error building site: "/opt/build/repo/content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset_phase_update-cluster-status.md:1:1": unmarshal failed: invalid character '{' looking for beginning of object key string

**Comment by neolit123** [comment]
> For some reason, I did not build that page (content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset_phase_update-cluster-status.md).

we removed the phase / command from source code in 1.23 and forgot to remove this section :\
https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase/#cmd-reset-phase-update-cluster-status

there shouldn't be a generated import page anymore as well.

**Comment by neolit123** [comment]
> > For some reason, I did not build that page (content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset_phase_update-cluster-status.md).
> 
> [we removed](https://github.com/kubernetes/kubernetes/commit/b9171aee20a2e3bd60cdf8ed3f269577b443eb4c) the phase / command from source code in 1.23 and forgot to remove this section : https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase/#cmd-reset-phase-update-cluster-status
> 
> there shouldn't be a generated import page anymore as well.

@kbhawkey @tengqm 
FYI, https://github.com/kubernetes/website/pull/30855

**Comment by tengqm** [comment]
It was my fault. I should have cleaned the existing `kubeadm*` files before copying the newly generated ones.
Now fixed.

**Comment by reylejano** [comment]
Changes look good, thank you everyone to confirm that kubeadm_reset_phase_update-cluster-status shouldn't be generated
Preview of updated pages:
- https://deploy-preview-30844--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_list/
- https://deploy-preview-30844--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_pull/
- https://deploy-preview-30844--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_init
- https://deploy-preview-30844--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_addon_all
- https://deploy-preview-30844--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_addon_coredns
- https://deploy-preview-30844--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_control-plane_all
- https://deploy-preview-30844--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_control-plane_apiserver
- https://deploy-preview-30844--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_join
- https://deploy-preview-30844--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset
- https://deploy-preview-30844--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply
- https://deploy-preview-30844--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan
- https://deploy-preview-30844--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase

/lgtm

**Pull request merged**